db/migrate/20171113153954_create_transactions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTransactions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :transactions do |t|
+    	t.date :transaction_date
```

Comment:
we don't need this field, since the `:created_at` field generated by AR will already track this for us 😅